locales/templates/app.pot

```diff
@@ -1446,6 +1446,45 @@ msgctxt "logo_text"
 msgid "Service provided by"
 msgstr ""
 
+#. Recommended news onboarding title and content.
+msgctxt "recommended_news_title"
+msgid "Recommended News For You"
+msgstr ""
+
+#, c-format
+msgctxt "recommended_news_content"
+msgid ""
+"Tap “Yes” to begin seeing news recommended for you by our AI news engine."
+" You can turn off recommendations and control your settings anytime in "
+"News-&gt;Settings. %1$s"
+msgstr ""
+
+#. Recommended news choose language
+msgctxt "recommended_news_preference_language"
+msgid ""
+"Choose Your Prefered\n"
```

Comment:
Should be "preferred".
Also, what does "preferred" refer to? Should be explained in localization comment so translators know how to  localize it.

Reply:
@Conjuror 
Change 
#. Recommended news choose language
to
#. Recommended news choose your preferred language

and Change
"Choose Your Prefered\n"
to
"Choose Your Preferred\n"

Reply:
DONE

Reply:
DONE